src/test/java/com/wikia/webdriver/testcases/portableinfoboxtests/PortableInfoboxTests.java

```diff
@@ -130,83 +133,70 @@ public void verifyCategoriesInTemplateInvocation() {
+
     theme.openSpecialDesignerPage(wikiURL);
     theme.selectTheme(1);
     theme.submitThemeSelection();
     article.open(PageContent.PORTABLE_INFOBOX01);
-    String newBackground = info.getBackgroundColor();
-    info.verifyChangedBackground(oldBackground, newBackground);
-
+    info.verifyChangedBackground(oldBackground, info.getBackgroundColor());
   }
 
-  @Test(groups = {"PortableInfoboxTests", "PortableInfoboxTests_012"})
+  @Test(groups = "PortableInfoboxTests_012")
   public void verifyOrderedAndUnorderedLists() {
-    ArticlePageObject article = new ArticlePageObject(driver);
-    article.open(PageContent.PORTABLE_INFOBOX02);
-    PortableInfoboxPageObject info = article.getInfoboxPage();
-    WebElement itemValue = info.getItemValue();
-    WebElement orderedListItem = info.getOrderedListItem(1);
-    info.compareFontSizes(itemValue, orderedListItem);
-    WebElement unorderedListItem = info.getUnorderedListElement(1);
-    info.compareFontSizes(itemValue, unorderedListItem);
+    new PortableInfoboxPageObject(driver)
+        .open(PageContent.PORTABLE_INFOBOX02)
+        .compareFontSizesBetweenItemValueAndOrderedListItem(1)
+        .compareFontSizesBetweenItemValueAndUnorderedListItem(1);
   }
 
-  @Test(groups = {"PortableInfoboxTests", "PortableInfoboxTests_013"})
+  @Test(groups = "PortableInfoboxTests_013")
   public void verifyInfoboxCategoryLink() {
-    ArticlePageObject article = new ArticlePageObject(driver);
-    article.open(PageContent.PORTABLE_INFOBOX01);
-    PortableInfoboxPageObject info = article.getInfoboxPage();
-    info.clickCategoryLink();
-    CategoryPageObject category = new CategoryPageObject(driver);
-    String categoryName = category.getCategoryName();
-    category.verifyCategoryPageTitle(categoryName);
+    new PortableInfoboxPageObject(driver)
+        .open(PageContent.PORTABLE_INFOBOX01)
+        .clickCategoryLink();
+    new CategoryPageObject(driver)
+        .verifyCategoryPageTitle();
   }
 
-  @Test(groups = {"PortableInfoboxTests", "PortableInfoboxTests_014"})
+  @Test(groups = "PortableInfoboxTests_014")
   public void verifyHorizontalGroupFontSize() {
-    ArticlePageObject article = new ArticlePageObject(driver);
-    article.open(PageContent.PORTABLE_INFOBOX01);
-    PortableInfoboxPageObject info = article.getInfoboxPage();
-    WebElement horizontalItemLabel = info.getHorizontalItemLabel();
-    WebElement itemLabel = info.getItemLabel();
-    info.compareFontSizes(horizontalItemLabel, itemLabel);
-    WebElement horizontalItemValue = info.getHorizontalItemValue();
-    WebElement itemValue = info.getItemValue();
-    info.compareFontSizes(horizontalItemValue, itemValue);
+    new PortableInfoboxPageObject(driver)
+        .open(PageContent.PORTABLE_INFOBOX01)
+        .compareFontSizesBetweenHorizontalItemLabelAndItemLabel()
+        .compareFontSizesBetweenHorizontalItemValueAndItemValue();
   }
 
+  // TODO: compress this test
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)